app/views/schools/participants/edit_name.html.erb

```diff
@@ -1,19 +1,19 @@
 <% content_for :before_content,
                govuk_back_link(text: "Back",
-                               href: schools_participant_edit_name_path(participant_id: @profile)) %>
+                               href: schools_participant_edit_name_path(participant_id: @profile,
+                                                                        selected_reason: @reason)) %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
 
     <span class="govuk-caption-xl"><%= @school.name %></span>
-    <%= form_for @profile.user, url: schools_participant_update_name_path, method: :put do |f| %>
+    <%= form_with model: @profile.user, scope: '', url: schools_participant_update_name_path, method: :put do |f| %>
       <%= f.govuk_error_summary %>
       <%= f.govuk_text_field :full_name,
-                             label: {
-                               text: "What should we edit #{@profile.full_name}’s name to?",
-                               tag: 'h1',
-                               size: 'xl'
-                             } %>
+                             value: nil,
+                             class: 'govuk-!-width-two-thirds',
```

Comment:
```suggestion
                             width: 'two-thirds',
```

Reply:
The prototype uses `govuk-!-width-two-thirds` class and if I change it to `two-thirds` the input box gets wider

Reply:
It's odd that it's wider with the arg than with the class being set by name, in the form builder it [_should_ be doing the same thing](https://github.com/DFE-Digital/govuk-formbuilder/blob/842cd45df3619d9b439a729d114a1caa9d6a3928/lib/govuk_design_system_formbuilder/traits/input.rb#L79).

Reply:
You are absolutely right. Sorry, my bad.
I just tried `class: 'two-thirds'` instead of `width: 'two-thirds'`

Reply:
Applied the same change to edit_email template